src/main/java/de/tum/in/www1/artemis/web/rest/ogparser/LinkPreviewResource.java

```diff
@@ -32,7 +35,7 @@ public LinkPreviewResource(LinkPreviewService linkPreviewService) {
      * POST /link-preview : link preview for given url.
      *
      * @param url the url to parse
-     * @return the LinkPreviewDTO containing the meta information
+     * @return the ResponseEntity with status 200 (OK) and with body the LinkPreviewDTO containing the meta information
```

Comment:
We should revert the comment as we don't return a response entity here. (You said this class is exempt from returning a response entity, so this is fine)
```suggestion
     * @return the LinkPreviewDTO containing the meta information
```